src/test/unit/algorithms/ConnectionsTest.cpp

```diff
@@ -509,78 +493,6 @@ namespace {
   }
 
   /**
-   * Creates a sample set of connections, and makes sure that getting the most
-   * active segment for a collection of cells returns the right segment.
-   */
-  TEST(ConnectionsTest, testMostActiveSegmentForCells)
```

Comment:
Are these deleted tests no longer application or do they need to be rewritten for the new code?